### PR TITLE
add invite link to nodejs slack

### PR DESCRIPTION
**Pull request by thescientist13**
1. added a link to the NodeJS Slack workspace to the Getting Involved page.  Followed language used on the OpenJS [collaboration page](https://openjsf.org/collaboration/)

### CODE REVIEW

**Review by Trott — DISMISSED** [review]
Everything else in the list is some form of entity with an official tie to the Node.js project/OpenJS Foundation, but NodeSlackers is its own independent thing. I don't think it belongs in this list.

Not relevant to this change in particular but since I'm looking at the list now: The Node.js Foundation isn't a thing anymore, so the items that mention that need to be reworded.

**Comment by Trott** [comment]
> Everything else in the list is some form of entity with an official tie to the Node.js project/OpenJS Foundation, but NodeSlackers is its own independent thing. I don't think it belongs in this list.

At the same time, I'm kinda at a loss as to where it should go so...
¯\\_(ツ)_/¯

**Comment by thescientist13** [comment]
Thanks for reviewing @Trott .  

I was having this same discussion with @ljharb in the OpenJS Foundation Slack, in the context of creating WG specific channels in the OpenJS Slack to continue the conversations from Node+JS Interactive Collab Summit , to which it was pointed out there is a NodeJS Slack too.  (but I couldn't find a link from the website)

> At the same time, I'm kinda at a loss as to where it should go so...

The link to this Slack does already exist in the https://github.com/nodejs/help repo as @ljharb pointed out to me, but I thought to help improve community contribution / collaboration, it might help to be on this page as well.  However, that would also introduce some duplication.  (maybe the Help repo should point to the website?)

> The Node.js Foundation isn't a thing anymore, 

Another option is to fold NodeJS slack into OpenJS slack, but that may be outside the scope of this PR.

**Comment by Trott** [comment]
Yeah, on balance I guess I'm for this. I do think this page needs a big ol' rethink, as does much of the site, though. But that's outside the scope of this PR.

**Review by ljharb — APPROVED** [review]
Unless there's a better place to put this, of course :-)

(The slack is adminned by a number of moderation team members and folks associated with node, so it's not *un*related)

**Review by MylesBorins — APPROVED** [review]
LGTM

I'm ok with this landing but will wait for someone else to land it

File: locale/en/get-involved/index.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ layout: contribute.hbs
 * [Node.js Everywhere](https://newsletter.nodejs.org) is the official Node.js Monthly Newsletter.
 * [Node.js Collection](https://medium.com/the-node-js-collection) is a collection of community-curated content on Medium.
 * The [Community Committee](https://github.com/nodejs/community-committee) is a top-level committee in the Node.js Foundation focused on community-facing efforts.
+* You can receive an invitation to join the NodeJS Slack workspace by visiting [https://www.nodeslackers.com/](https://www.nodeslackers.com/).  Once you join the NodeJS workspace, you can participate in any of the public channels.  If you are new to Slack, you can view a full list of available channels by clicking the “Channels” link on the left.
```
Comment (Trott) [inline locale/en/get-involved/index.md]: Maybe keep it short and simple?

```suggestion
* [Node Slackers](https://www.nodeslackers.com/) is a Node.js-focused Slack community.
```

**Pull request merged**